File: vapor-app/src/main/java/com/tevinjeffrey/vapor/ui/login/LoginException.java
package com.tevinjeffrey.vapor.ui.login;

public class LoginException extends Exception {
    private int code;
    public LoginException() {
    }

    public LoginException(String detailMessage) {
        super(detailMessage);
    }

    public LoginException(String detailMessage, Throwable throwable) {
        super(detailMessage, throwable);
    }

    public void setCode(int code) {
        this.code = code;
    }

    public int getCode() {
        return code;
    }
}

File: vapor-app/src/main/java/com/tevinjeffrey/vapor/ui/login/LoginException.kt
package com.tevinjeffrey.vapor.ui.login

class LoginException : Exception {
    var code: Int = 0

    constructor() {
    }

    constructor(detailMessage: String) : super(detailMessage) {
    }

    constructor(detailMessage: String, throwable: Throwable) : super(detailMessage, throwable) {
    }
}
